docs/guides/managed-backup/dedicated-backup-namespace-auto-backup/index.md

```diff
@@ -0,0 +1,407 @@
+---
+title: Define Cluster Wide Backup Policy using Stash Auto-backup
+description: A guide on how to define cluster wide backup policy using Stash auto-backup.
+menu:
+  docs_{{ .version }}:
+    identifier: managed-backup-dedicated-backup-namespace-auto-backup
+    name: Define Cluster Wide Backup Policy using Stash Auto-backup
+    parent: managed-backup
+    weight: 30
+product_name: stash
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+# Define Cluster Wide Backup Policy using Stash Auto-backup
+
+This guide will show you how you can define cluster wide backup policy using Stash Auto-backup.
+
+## Before You Begin
+
+- At first, you need to have a Kubernetes cluster, and the `kubectl` command-line tool must be configured to communicate with your cluster. If you do not already have a cluster, you can create one by using [kind](https://kind.sigs.k8s.io/docs/user/quick-start/).
+
+- Install `Stash` in your cluster following the steps [here](/docs/setup/README.md).
+
+- You should be familiar with the following `Stash` concepts:
+  - [BackupConfiguration](/docs/concepts/crds/backupconfiguration.md)
+  - [BackupSession](/docs/concepts/crds/backupsession.md)
+  - [Repository](/docs/concepts/crds/repository.md)
+
+Here, we are going to take backup from the `prod-1`, `prod-2` and `prod-3` namespaces and manage the backup from `backup` namespace using Auto Backup.
```

Comment:
We are going to take backup from the `prod-1`, `prod-2` and `prod-3` namespaces. We want to create our backup resources in `backup` namespace.

Let’s create the mentioned namespaces: